docs/tab-widgets/distributed-trace-receive.asciidoc

```diff
@@ -33,6 +33,17 @@ transaction := apm.DefaultTracer.StartTransactionOptions("GET /", "request", opt
 // ***************************************************
 // ***************************************************
 
+// tag::ios[]
+
+experimental::[]
+
+_coming soon_
+
+// end::ios[]
```

Comment:
@bryce-b do you want to fill these out in this PR or in a follow-up? [Link](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/distributed-tracing.html#_add_the_traceparent_header_to_outgoing_requests).

Reply:
incoming requests isn't really applicable to mobile.